scripts/extract-release-changelog.py

```diff
@@ -14,6 +14,13 @@ def get_tag():
     revision = p.stdout.decode().strip()
     return revision
 
+def get_prev_tag():
+    rev = subprocess.run(['git', 'rev-list', '--tags', '--skip=0', '--max-count=1'], capture_output=True)
+    rev = rev.stdout.decode().strip()
```

Comment:
since the first `rev` is subprocess object, how about rename it?
this is just a suggestion.